docker-compose.yaml

```diff
@@ -0,0 +1,67 @@
+version: "3.2"
+
+services:
+  starknet:
+    image: ghcr.io/keep-starknet-strange/madara:v0.1.0-testnet-sharingan-beta.7.1.experimental.4@sha256:baff5a0aa1075d0f84b83ed366f56928b4a6a0c3316762df14e8b1656f784729
+    ports:
+      - "9615:9615"
+      - "9944:9944"
+      - "30333:30333"
+    command:
+      - "--rpc-external"
+      - "--rpc-methods=unsafe"
+      - "--dev"
+    networks:
+      - internal
+
+  kakarot-deployer:
+    image: ghcr.io/kkrt-labs/kakarot/deployer:latest
+    environment:
+      - ACCOUNT_ADDRESS=0x3 
+      - PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d 
+      # Custom RPC URL for docker
+      - RPC_URL=http://starknet:9944
+      - EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 
+    volumes:
+      # Since STARKNET_NETWORK is not set, deployments will be saved without network folder
+      - deployments:/app/kakarot/deployments
+    depends_on:
+      - starknet
+    restart: on-failure
+    networks:
+      - internal
+
+  kakarot-rpc:
+    image: ghcr.io/kkrt-labs/kakarot-rpc/node:latest
+    ports:
+      - 3030:3030
+    # Override entrypoint to set custom env variables
+    entrypoint:
+      - "/bin/sh"
+      - "-c"
+      - |
+        export KAKAROT_ADDRESS=$(jq -r '.kakarot.address' ./deployments/deployments.json) && \
+        export PROXY_ACCOUNT_CLASS_HASH=$(jq -r '.proxy' ./deployments/declarations.json) && \
+        echo Proxy account class hash: ${PROXY_ACCOUNT_CLASS_HASH} && \
+        echo Kakarot address: ${KAKAROT_ADDRESS} && \
+        /usr/bin/tini -- /usr/local/bin/kakarot-rpc
```

Comment:
`tini` is meant to be the entrypoint of the image, see [tini doc](https://github.com/krallin/tini)

Following on our previous comment regardig $GITHUB_OUTPUT, it think that a legit step could be using the `alpine` image to convert the deployments into a .env file used in `kakarot-rpc: env_file` section, or just to overwrite the `kakarot-deployer image to create such a file at the end (I don't like this one because it makes our file break if the deployer changes, but not such a big deal I think)

Reply:
Saw this on the [tini doc](https://github.com/krallin/tini#existing-entrypoint) about existing entrypoints and tried it out and it works for our usecase 🤓 let me know what you think

![image](https://github.com/kkrt-labs/kakarot-rpc/assets/35031356/7b81ac29-00f4-4d10-85b3-af76b03ce62a)